src/decryption/decryption.ts

```diff
@@ -134,37 +125,39 @@ function _deriveKeyWithOptions(
   return derivedKeyOptions.deriveKey(key, encodingVersion);
 }
 
+/**
+ * @deprecated This method should be replaced by
+ * decryptWithKeyUsingArtefacts method. This method convert give bytes to utf8 string
+ */
 export function decryptStringWithKeyUsingArtefacts(
   key: string,
   encryptedData: any,
   strategy: CipherStrategy,
   { iv, at, ad }: IEncryptionOptions
 ) {
-  return decryptWithKeyUsingArtefacts(key, encryptedData, strategy, { iv, at, ad }, 'utf8');
+  const result = decryptWithKeyUsingArtefacts(key, encryptedData, strategy, { iv, at, ad });
+  return result ? bytesToUtf8(result) : null;
 }
 
+/**
+ * @deprecated This method should be replaced by
+ * decryptWithKeyUsingArtefacts method. This method convert give bytes to string
+ */
 export function decryptBinaryWithKeyUsingArtefacts(
   key: string,
   encryptedData: any,
   strategy: CipherStrategy,
   { iv, at, ad }: IEncryptionOptions
 ) {
-  return decryptWithKeyUsingArtefacts(key, encryptedData, strategy, { iv, at, ad }, 'raw');
+  const result = decryptWithKeyUsingArtefacts(key, encryptedData, strategy, { iv, at, ad });
+  return result ? bytesToBinary(result) : null;
```

Comment:
I think either rename this to `decodeByteString` or just use `util.binary.raw.decode()` directly since we're internal here so there's nothing wrong with just using raw node-forge methods.